ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEthSendTransactionIntegrationTest.java

```diff
@@ -386,6 +386,7 @@ void signSendTransactionWhenContract() {
 
   @Test
   void signSendTransactionWhenContractWithLongChainId() throws Exception {
+    cleanUpInstance();
     setupEthSigner(4123123123L);
```

Comment:
If you're encountering resourcing issues, setting up another EthSigner does not look like sensible approach (as an EthSigner is setup by the IntegrationBase).

Reply:
Moving these tests that require a different setup to classes that do not extend from the base (with it's overhead of another EthSigner) would be a better approach IMHO.

Reply:
Excellent suggestion. 👍